### PR TITLE
[Finishes #95930328] Allow users to chose assets in code mode from a paramter tooltip

### PR DESCRIPTION
**Pull request by joshlory**
The other part of Flow _#_2 for the Manage Assets pipeline:

asset_tooltip

Allows users to choose assets from tooltips while writing code in Applab.

Will conflict with https://github.com/code-dot-org/code-dot-org/pull/2770.

### CODE REVIEW

File: apps/src/blockTooltips/DropletAutocompleteParameterTooltipManager.js [hunk 1]
```diff
@@ -89,6 +89,19 @@ DropletAutocompleteParameterTooltipManager.prototype.updateParameterTooltip_ = f
...
     event.stopPropagation();
   }.bind(this));
+
+  var chooseAsset = tooltipInfo.parameterInfos[currentParameterIndex].assetTooltip;
+  if (chooseAsset) {
+    var chooseAssetLink = $(cursorTooltip.tooltipster('elementTooltip')).find('.tooltip-choose-link > a')[0];
+    dom.addClickTouchEvent(chooseAssetLink, function(event) {
+      cursorTooltip.tooltipster('hide');
+      chooseAsset(function(filename) {
+        console.log(filename);
```
Comment (bcjordan) [inline apps/src/blockTooltips/DropletAutocompleteParameterTooltipManager.js]: from debugging?

Comment (Bjvanminnen) [inline apps/src/blockTooltips/DropletAutocompleteParameterTooltipManager.js]: intentionally left in?

**Comment by Bjvanminnen** [comment]
lgtm

**Comment by bcjordan** [comment]
lgtm!

**Pull request merged**